public/app/services/apps.ts

```diff
@@ -96,6 +104,8 @@ export async function fetchApps(): Promise<
         '__type__',
         '__name__',
       ],
+      start: fromMs,
+      end: toMs,
```

Comment:
Please consider enjoying this syntactic sugar. 
```suggestion
      start: fromMs || 0,
      end: toMs || 0,
```

If anything before `||` resolves to false, it will default to what follows, which will allow you to remove the if checks above.

Reply:
Ah nice! Added in https://github.com/grafana/pyroscope/pull/2482/commits/cd03f458510c08cd278fc6c5bd89518e1a02a681